graphics/opengl/shader.cpp

```diff
@@ -155,7 +155,6 @@ GLuint Shader::createCompatShader(const char *shaderSource, GLenum shaderType, c
 	GLchar versionSource[20];
 	if (OpenGLContext.type == kContextGLES2) {
 		switch(compatGLSLVersion) {
-			case 100:
```

Comment:
There is no difference in removing this, since `compatGLSLVersion` will be set to 100 below anyway